### PR TITLE
Add bash-firefighters team

### PR DESCRIPTION
**Pull request by cblecker**
:bash_fire:

@kubernetes/sig-testing anyone else want in on this?
cc: @fejta @BenTheElder @cjwagner @ixdy @stevekuznetsov

### CODE REVIEW

**Comment by cblecker** [comment]
by popular demand https://github.com/kubernetes/repo-infra/issues/97#issuecomment-456841978

**Comment by ixdy** [comment]
can't tell if serious

**Comment by cblecker** [comment]
This is a legitimate PR that will create a legitimate team haha

**Comment by BenTheElder** [comment]
Looking forward to the bash BOF at the next KubeCon 😉

**Comment by stevekuznetsov** [comment]
:bash_fire"

**Comment by nikhita** [comment]
I think @sttts should be on the team as well (sorry Stefan! :laughing: )

**Comment by spiffxp** [comment]
Does it need to be sig-testing? I feel like there are many dirs that have bash that are not just under sig-testing's provenance

**Comment by spiffxp** [comment]
Common dirs that need this expertise:
- kubernetetes/repo-infra - a contribex subproject
- kubernetes/kubernetes/hack - doesn't actually belong to any one sig
- kubernetes/kubernetes/build - same (but probably should?)

File: config/kubernetes/sig-testing/teams.yaml [hunk 1]
```diff
@@ -37,3 +37,14 @@ teams:
     - cjwagner
     - mkumatag
     privacy: closed
+  sig-testing-bash-reviews:
```
Comment (spiffxp) [inline config/kubernetes/sig-testing/teams.yaml]: If you drop sig-testing from the name, then I'm fine with it, though I still think this is a volunteer effort at best and may not improve the situation.  All of these folks are pretty oversubscribed as it is

File: config/kubernetes/org.yaml [hunk 2]
```diff
@@ -811,6 +811,18 @@ teams:
...
+  bash-firefighters:
+    description: "Folks with expertise in bash reviews"
+    maintainers:
+    - cblecker
+    - fejta
+    members:
+    - BenTheElder
+    - cjwagner
+    - ixdy
+    - stevekuznetsov
```
Comment (spiffxp) [inline config/kubernetes/org.yaml]: _...sighs_.... and my axe

put me in coach

**Pull request merged**